src/api/commands.ts

```diff
@@ -31,136 +31,183 @@ export class AccountInfo extends Command {
         checkField(data, "flags", "object", "undefined");
         checkField(data, "friends", "array", "undefined");
         checkField(data, "rooms", "array", "undefined");
-        
+
         return { ...data, created_at: new Date(data.created_at) };
     }
 }
 
 export class AccountChange extends Command {
     constructor(account_id: string, changes: { label?: string, name?: string, surname?: string }) {
-        super(["account-accountChange", {account_id, changes: JSON.stringify(changes)}]);
+        super(["account-accountChange", { account_id, changes: JSON.stringify(changes) }]);
+
+        checkType("account_id", account_id, "string");
+        checkType("changes", changes, "string");
     }
 
-    parse(status: number, data: IAccount) {
-        if(status !== 200) return data;
+    parse(status: number, data: any) {
+        if (status !== 204) return data;
 
-        // TODO: Ensure types.
-        
         return data;
     }
 }
 
 export class AccountList extends Command {
     constructor(included?: string[], volume?: number) {
-        super(["account-accountList", { volume, included }]);
+        super(["account-accountList", { volume: !!volume, included: !!included }]);
```

Comment:
`volume` has to be a number, not a boolean. Same goes to included, it has to be a list of strings, not a boolean. 
```suggestion
        super(["account-accountList", { volume: +volume, included }]);
```

Here is an example of boolean parsing shorthand:
```ts
const list1 = ["foo", "bar"];
const list2 = [];
const numb1 = 15;
const numb2 = 0;
const str1 = "test";
const str2 = "";

console.log(!!list1); //  true  -- array is always truthy
console.log(!!list2); //  true  -- array is always truthy
console.log(!!numb1); //  true  -- positive number
console.log(!!numb2); //  false  -- number is falsy (0 and below)
console.log(!!str1); //  true  -- string has content
console.log(!!str2); //  false  -- string is falsy (empty)

// !! === Boolean()
```

Using just one exclamation mark (!) will return the opposite boolean.